nav2_bt_navigator/README.md

```diff
@@ -56,6 +56,8 @@ children till the second one succeeds. Then it will tick the first three till th
 third succeeds and so on, till there are no more children. This will return RUNNING,
 till the last child succeeds, at which time it also returns SUCCESS. If any child
 returns FAILURE, all nodes are halted and this node returns FAILURE.
+* RoundRobin: This is a custom control node introduced to the Behavior Tree. When this node is ticked, it will tick the first child until it returns SUCCESS or FAILURE. If the child returns either SUCCESS or FAILURE, it will tick its next child. Once the node reaches the last child, it will restart ticking from the first child.
+
```

Comment:
This sounds an awful lot like a sequence. I'd add another paragraph to explain how this is different from a sequence

Reply:
The main difference is when a child returns Failure the sequence node will return Failure and it will not tick the next child.